app/_includes/md/kic/deploy-kic.md

```diff
@@ -1,43 +1,49 @@
 ## Deploy the {{site.kic_product_name}} {#deploy-kic}
 
-Deploy the {{site.kic_product_name}} using `kubectl`:
+You can deploy the {{site.kic_product_name}} on minikube using `kubectl` or `Helm Charts`:
 
 {% if_version gte:2.8.x %}
+{% navtabs codeblock %}
+{% navtab kubectl %}
 ```bash
 $ kubectl create -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{ include.version }}/deploy/single/all-in-one-dbless.yaml
 ```
-The output is similar to:
+{% endnavtab %}
+{% navtab Helm Charts %}
 ```bash
-namespace/kong created
-customresourcedefinition.apiextensions.k8s.io/ingressclassparameterses.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/kongclusterplugins.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/kongconsumers.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/kongingresses.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/kongplugins.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/tcpingresses.configuration.konghq.com created
-customresourcedefinition.apiextensions.k8s.io/udpingresses.configuration.konghq.com created
-serviceaccount/kong-serviceaccount created
-role.rbac.authorization.k8s.io/kong-leader-election created
-clusterrole.rbac.authorization.k8s.io/kong-ingress created
-clusterrole.rbac.authorization.k8s.io/kong-ingress-gateway created
-clusterrole.rbac.authorization.k8s.io/kong-ingress-knative created
-rolebinding.rbac.authorization.k8s.io/kong-leader-election created
-clusterrolebinding.rbac.authorization.k8s.io/kong-ingress created
-clusterrolebinding.rbac.authorization.k8s.io/kong-ingress-gateway created
-clusterrolebinding.rbac.authorization.k8s.io/kong-ingress-knative created
-service/kong-admin created
-service/kong-proxy created
-service/kong-validation-webhook created
-deployment.apps/ingress-kong created
-deployment.apps/proxy-kong created
-ingressclass.networking.k8s.io/kong created
+$ helm repo add kong https://charts.konghq.com
+$ helm repo update
+
+# Helm 3
+$ helm install kong/kong --generate-name
 ```
+{% endnavtab %}
+{% endnavtabs %}
+
 {% endif_version %}
+
+
 {% if_version lte:2.7.x %}
+{% navtabs codeblock %}
+{% navtab kubectl %}
 ```bash
 $ kubectl create -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{ page.version }}/deploy/single/all-in-one-dbless.yaml
 ```
-The output is similar to:
+{% endnavtab %}
+{% navtab Helm Charts %}
+```bash
+$ helm repo add kong https://charts.konghq.com
+$ helm repo update
+
+# Helm 3
+$ helm install kong/kong --generate-name
```

Comment:
Using `--generate-name` means that any future commands are unlikely to work. We should use a specific release name

Reply:
I see that you already updated this to ```bash
helm repo add kong https://charts.konghq.com
helm repo update
helm install kongdocs kong/kong
```